me.jtheoof.swappy.metainfo.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
+    <id>me.jtheoof.swappy</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>MIT</project_license>
```

Comment:
Add developer_name

Reply:
Isn't it deprecated? I used the `<developer>` tag.

https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-developer_name

Reply:
Flathub doesn't use libappstream. There's a check for missing developer name tag that's going to be turned on next year.